shim/src/accessors.js

```diff
@@ -86,3 +83,5 @@ export function repairAccessors(unsafeRec) {
     }
   });
 }
+
+export const repairAccessorsShim = `${repairAccessors} repairAccessors();`;
```

Comment:
1. We could be leaking a global function called `repairAccessors()`. Why not do
```js
export const repairAccessorsShim = `(${repairAccessors})();`;
```	
2. Let's add a note saying that `repairAccessors` is constructed to be compatible with `repairAccessorsShim`. In particular, all declarations need to be in the scope of repairAccessors().

Reply:
Oh, yeah, sounds much better. I'll add that.

Reply:
is the comment on line 26 sufficient?